spec/factories/questions.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :question do
+    heading { "MyString" }
+    statement { "MyText" }
+    category { nil }
+    difficulty_level { nil }
+    test_data { "" }
+    circuit_boilerplate { "" }
+  end
```

Comment:
Use `Faker` . Ref- https://github.com/faker-ruby/faker?tab=readme-ov-file#usage

For test_data and circuit_boilerplate, we may put a blank JSON instead of blank string